### PR TITLE
updated whatsnew with normalize-by-median broken-paired updates

### PR DESCRIPTION
**Pull request by bocajnotnef**
- [x] Is it mergeable?
- [x] Did it pass the tests?
- [x] If it introduces new functionality in scripts/ is it tested?
  Check for code coverage with `make clean diff-cover`
- [x] Is it well formatted? Look at `make pep8`, `make diff_pylint_report`,
  `make cppcheck`, and `make doc` output. Use `make format` and manual
  fixing as needed.
- [x] Did it change the command-line interface? Only additions are allowed
  without a major version increment. Changing file formats also requires a
  major version number increment.
- [x] Is it documented in the ChangeLog?
  http://en.wikipedia.org/wiki/Changelog#Format
- [x] Was a spellchecker run on the source code and documentation after
  changes were made?
- [x] Is the Copyright year up to date?

@mr-c @ctb @luizirber Merge, please?

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,7 @@
+2015-07-01  Jaob Fenton  <bocajnotnef@gmail.com>
+
+   * doc/whats-new-2.0.rst: added in norm-by-med broken paired updates
```
Comment (ctb) [inline ChangeLog]: please use the full script name - that way we can grep for it.

File: doc/whats-new-2.0.rst [hunk 2]
```diff
@@ -3,6 +3,19 @@
...
 ########################
 
+
+Script Updates
+==============
+
+Normalize by median supports broken-paired input
+------------------------------------------------
+
+Normalize-by-median now supports mixed paired and unpaired input. Behavior can
```
Comment (ctb) [inline doc/whats-new-2.0.rst]: `normalize-by-median.py`

File: doc/whats-new-2.0.rst [hunk 3]
```diff
@@ -3,6 +3,19 @@
 What's New In khmer 2.0?
 ########################
 
+
+Script Updates
+==============
+
+Normalize by median supports broken-paired input
```
Comment (ctb) [inline doc/whats-new-2.0.rst]: As long as I'm micromanaging :)

"Digital normalization script now supports mixed paired and unpaired read input." And then, below, say "(or broken-paired input)".  broken-paired is not standard bioinfo lingo.

File: doc/whats-new-2.0.rst [hunk 4]
```diff
@@ -3,6 +3,19 @@
...
+------------------------------------------------
+
+Normalize-by-median now supports mixed paired and unpaired input. Behavior can
+be forced to either treat all reads as singletons or to require all reads be
+properly paired using --force-single or --paired, respectively. If --paired is
+set, -u can be used to include a file of unpaired reads.
```
Comment (ctb) [inline doc/whats-new-2.0.rst]: "The unpaired reads will be examined after all of the other sequence files."

**Comment by mr-c** [comment]
@bocajnotnef ping

**Comment by bocajnotnef** [comment]
@mr-c Pong!

**Comment by ctb** [comment]
LGTM. Thanks!

**Pull request merged**